mono_repo/lib/src/raw_config.g.dart

```diff
@@ -8,25 +8,25 @@ part of 'raw_config.dart';
 
 RawConfig _$RawConfigFromJson(Map json) {
   return $checkedNew('RawConfig', json, () {
-    $checkKeys(json, allowedKeys: const ['dart', 'stages', 'cache']);
-    var val = new RawConfig(
+    $checkKeys(json, allowedKeys: ['dart', 'stages', 'cache']);
```

Comment:
hmm, this is a minor behavior change. Which change in json_serializable does this correspond to? Should we revisit it? I see a few missing `const` which do change the semantics.

Reply:
ack! https://github.com/dart-lang/json_serializable/issues/288